lib/ui/settings/change_password/change_password_page.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:pro_mobile/ui/settings/change_password/change_password_view_modal.dart';
+import 'package:provider/provider.dart';
+import 'package:pro_mobile/constants/app_colors.dart';
+import 'package:pro_mobile/ui/widgets/custom_text.dart';
+import 'package:pro_mobile/ui/widgets/form/custom_text_input_2.dart';
+import 'package:pro_mobile/ui/widgets/padded_container.dart';
+
+class ChangePasswordScreen extends StatelessWidget {
+  const ChangePasswordScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ChangeNotifierProvider<ChangePasswordModalView>(
```

Comment:
We wrap all our UI pages that need business logic with "BaseView". So, lets consider replacing the "ChangeNotifierProvider" with the already existing BaseView